read-fonts/src/lib.rs

```diff
@@ -21,6 +21,7 @@ mod table_ref;
 pub mod tables;
 #[cfg(feature = "traversal")]
 pub mod traversal;
+pub mod variations;
```

Comment:
so I've been wondering if actually *all* tables (including layout and variations) should live in the *tables* module? It has a certain clarity, e.g. It makes clear that everything that is *not* in the tables module is sort of core implementation stuff.

This is not really related to this PR, but this feels like a good time to discuss it.

Reply:
I was wondering the same thing. I think all should live in tables and the layout/variations modules should be moved there as well. I can't think of any reason to special case them and they'll stand out anyway due to the longer names.

Reply:
+1 to all in tables